configure.ac

```diff
@@ -238,6 +238,39 @@ AX_VALGRIND_DFLT([sgcheck], [off])
 # Add a target to run testsuite with valgrind.
 AX_VALGRIND_CHECK()
 
+# Requirements for running test-crypt-kat-gen.py.
+ENABLE_CRYPT_KAT_GEN="no"
+PYTHON_BIN="python3 python"
+
+for ver in `seq 0 1 10`; do
+  PYTHON_BIN="python3.$ver $PYTHON_BIN"
```

Comment:
Shouldn't this be `seq 6 1 10` since we already know 3.5 and lower won't work?

Also I suggest `PYTHON_BIN` should be called `PYTHON_CANDIDATES` instead.  `PYTHON_BIN` feels too likely to collide with some build robot's environment variables.

Reply:
Fixed.